sonaranalyzer-dotnet/rspec/cs/S5773_c#.html

```diff
@@ -17,7 +17,7 @@ <h2>Noncompliant Code Example</h2>
 href="https://docs.microsoft.com/en-us/dotnet/api/system.runtime.serialization.formatters.binary.binaryformatter?view=netframework-4.8">BinaryFormatter</a>,
 <a
 href="https://docs.microsoft.com/en-us/dotnet/api/system.runtime.serialization.netdatacontractserializer?view=netframework-4.8">NetDataContractSerializer
-</a>, <a href="https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.objectstateformatter?view=netframework-4.8">ObjectStateFormatter</a>, <a
```

Comment:
just to double check - is this ok to be removed? is it according to the implementation?

Reply:
Yes. This formatter is not using a binder. It has the property since it implements `IFormatter` but the property set will just ignore the received value.